haystack/utils/doc_store.py

```diff
@@ -4,53 +4,44 @@
 
 
 logger = logging.getLogger(__name__)
+ELASTICSEARCH_CONTAINER_NAME = "elasticsearch"
+OPENSEARCH_CONTAINER_NAME = "opensearch"
+MILVUS1_CONTAINER_NAME = "milvus1"
+WEAVIATE_CONTAINER_NAME = "weaviate"
 
 
-def launch_es(sleep=15):
+def launch_es(sleep=15, delete_existing=False):
     # Start an Elasticsearch server via Docker
 
     logger.debug("Starting Elasticsearch ...")
-    status = subprocess.run(
-        ['docker run -d -p 9200:9200 -e "discovery.type=single-node" elasticsearch:7.9.2'], shell=True
-    )
-    if status.returncode:
-        logger.warning(
-            "Tried to start Elasticsearch through Docker but this failed. "
-            "It is likely that there is already an existing Elasticsearch instance running. "
-        )
-    else:
-        time.sleep(sleep)
-
-
-def launch_open_distro_es(sleep=15):
-    # Start an Open Distro for Elasticsearch server via Docker
-
-    logger.debug("Starting Open Distro for Elasticsearch ...")
+    if delete_existing:
+        _ = subprocess.run([f"docker rm --force {ELASTICSEARCH_CONTAINER_NAME}"], shell=True, stdout=subprocess.DEVNULL)
     status = subprocess.run(
         [
-            'docker run -d -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" amazon/opendistro-for-elasticsearch:1.13.2'
+            f'docker run -d -p 9200:9200 -e "discovery.type=single-node" --name {ELASTICSEARCH_CONTAINER_NAME} elasticsearch:7.9.2'
```

Comment:
I like the idea of not creating new containers all the time when running our tutorials. However, some users might at first not understand why their results differ if they execute tutorials multiple times (and the docker container caries over some indexed documents). Even worse, it could well be that we delete some data that users wanted to keep. One idea to mitigate this would be to use index names throughout the tutorials. For example, in https://colab.research.google.com/github/deepset-ai/haystack/blob/master/tutorials/Tutorial1_Basic_QA_Pipeline.ipynb let's initialize 
`document_store = ElasticsearchDocumentStore(host="localhost", index="document")` with `index="tutorial1_document_index"`. What do you think?

Reply:
It would clearly be an improvement to have these separate indexes in the tutorials. However in the current impl if there is already a container running, overwriting the index would happen too.